core/models.py

```diff
@@ -150,12 +151,13 @@ def activate_user(self, activation_token):
 
             self.is_active = True
             self.save()
+
             # valid_user is the routing followed by name, email and id
-            # mq_newuser('user.new', json.dumps({
-            #    'name': self.name,
-            #    'email': self.email,
-            #    'id': self.id
-            # }))
+            mq_newuser('user.new', json.dumps({
+               'name': self.name,
+               'email': self.email,
+               'id': self.id
```

Comment:
```suggestion
               'id': self.id
```
While running this I had issues on the PaaS side. PaaS won't create a profile because it expects a value of gcID instead of id.